Appraisals

```diff
@@ -1,9 +1,46 @@
 # frozen_string_literal: true
 
 appraise 'rails-6-1' do
-  gem 'rails', '~> 6.1.0'
+  gem 'actionmailer', '~> 6.1.0'
+  gem 'actionpack', '~> 6.1.0'
+  gem 'activejob', '~> 6.1.0'
+  gem 'activerecord', '~> 6.1.0'
+  gem 'activesupport', '~> 6.1.0'
```

Comment:
FYI, you could probably just specify `railties` here and the others would fall in line.